### DISCUSSION TITLE
Login issue in laravel 9.x with filament

### DISCUSSION THREAD

**Discussion by mnafullstack** [filamentphp/filament#3187] [General]
I have just installed the filament admin panel package with my fresh laravel project. If I run the project by "php artisan serve" the application logins successfully. But when I try to run the project using localhost/myproject/admin/login it runs fully but it doses not logins to the application and returns to the same login page after clicking on the login page with a question mark "?" in the url.

It also does not show any kind of errors. My env debug is set to true already.

I have tried to make it live on a server but on a live server, it is also not working. Does anybody have any idea how to diagnose the issue?

**Comment by danharrin** [comment]
Any console errors?

**Reply by mnafullstack** [reply]
Yes there is console error, it is saying that the livewire.js file could not be found. But when I run the application through php artisan serve, the application runs successfully and there is not any error in console then.

**Reply by danharrin** [reply]
Well livewire.js makes the entire admin panel work, so a 404 will break it.

Your issue is a bad server configuration.

https://github.com/livewire/livewire/issues/242#issuecomment-948054946
